RecoMET/METProducers/interface/CSCHaloDataProducer.h

```diff
@@ -151,6 +154,9 @@ class CSCHaloDataProducer : public edm::stream::EDProducer<> {
     //Muon-Segment Matching
     MuonSegmentMatcher *TheMatcher;
 
+    //Muon Service
+    MuonServiceProxy* TheService;
```

Comment:
This was removed in #10482

If it's just a forward port artifact, please remove here